gulp/tasks/copy.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
+var config = require( '../config' ).copy;
```

Comment:
Should be `var copy`, see https://nodejs.org/api/child_process.html for example of naming style.